types/cesium/index.d.ts

```diff
@@ -3024,7 +3029,7 @@ declare namespace Cesium {
     abstract class Property {
         readonly isConstant: boolean;
         readonly definitionChanged: Event;
-        getValue(time: JulianDate, result?: any): any;
```

Comment:
According to the documentation, `time` is not optional for all `Property`: https://cesium.com/docs/cesiumjs-ref-doc/Property.html?classFilter=Property